app/services/LocationService.js

```diff
@@ -191,24 +192,27 @@ export default class LocationServices {
     if (!isDeviceGpsEnabled) {
       return {
         canTrack: false,
-        reason: this.DEVICE_LOCATION_OFF,
+        reason: DEVICE_LOCATION_OFF,
         hasPotentialExposure,
         isRunning,
       };
     }
-
-    if (!isAppGpsEnabled) {
+    /* 
+      On Android: isAppGpsEnabled = 0 means Never Use Location
```

Comment:
Should these values be set as constants?
something like:
```
IOS_NEVER_USE_LOCATION = 0
```

Reply:
👍